pytest_answer.ini

```diff
@@ -0,0 +1,15 @@
+[pytest]
+local-dir=/home/travis/build/yt-project/yt/answer-store
+# Ignore those files that have tests that contain nose-specific code
+# For the first run, we need to create the gold standard answers
+# NOT running the big data tests for now
+addopts = 
+    -s
+    -v
+    --with-answer-testing
+    -rs
+    --answer-store
+    --force-overwrite
```

Comment:
@jcoughlin11 Correct me if I'm wrong, but does it mean that currently tests are overwriting the results with every run instead of comparing the answers?

Reply:
@Xarthisius Whoops! Yes, you're right, they are overwriting! That's my fault! I needed to generate a set of answers and then I thought I deleted those options, but apparently I did not. I'll delete them and re-push (though, locally, I'm having an issue with the hashes of ImageArrays)